version 4/00 intro/05 jquery/README.md

```diff
@@ -5,29 +5,30 @@ basic pillars of web development, consuming third party libraries.
 
 In this demo we will install a legacy library (jquery) via npm, define it as global, and use it. Finally we will end up creating a separate bundle for libraries.
 
-We will start from sample _00 Intro/03 Output_.
+We will start from sample _00 intro/_05 jquery_.
```

Comment:
I think we take as starting point 03 Output sample then we code it step by step and we get 05 Jquery?